book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author
+
+  has_many :rentals
+  has_many :persons, through: :rentals
```

Comment:
- [ ] The `has_many` methods here are undefined and they prevent me from creating a Book object, kindly remove them as highlighted by the previous reviewer.